.gitignore

```diff
@@ -31,3 +31,5 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+
+/coverage
```

Comment:
Double check me, I though it was 'coverage/'

Reply:
I typically utilize the syntax above and it seems like the coverage files did not come through with the PR.